arches_for_science/pkg/graphs/resource_models/Group.json

```diff
@@ -7837,20 +7837,20 @@
                         "descriptor_types": {
                             "description": {
                                 "nodegroup_id": "32db94f8-c05e-11e9-a2c1-a4d18cec433a",
-                                "string_template": "<Statement_content>"
+                                "string_template": "<statement_content>"
```

Comment:
These now need to be node aliases rather than node names. I could make the lookup case-insensitive, I suppose?

Reply:
That would work in many cases, but a node alias can differ from the node name in other ways, generally because an alias must be unique to a graph and a name only must be unique to its siblings. So, I think it's good to be explicit about changing from names to aliases as you've done here.